client/app/services/widget.js

```diff
@@ -146,14 +146,16 @@ function WidgetFactory($http, $location, Query) {
         }
         this.queryResult = this.getQuery().getQueryResult(maxAge);
 
-        this.queryResult.toPromise()
+        return this.queryResult.toPromise()
```

Comment:
I think this has been playing with me for a while (in here and in the Widget Migration).

![image](https://user-images.githubusercontent.com/3356951/66090671-ce272500-e55a-11e9-8680-015dfd13668f.png)

It was returning the queryResult before updating `this.data`, which was a weird result when I expected the widgets to render an error and they were not doing it